website/blog/2022-09-06/index.md

```diff
@@ -49,13 +50,15 @@ The theme for this week is Azure Functions. We'll talk about ...
 
 
 ## Azure Functions and Serverless explained.
+
 Azure Functions is a serverless compute service that enables you to run code on-demand without explicitly provisioning or managing the underlying infrastructure. 
 
 Developing code that will run in a serverless environment departs from the traditional ASP.NET WebAPI technique of creating controllers with actions. Instead, as the name implies, we make functions as small snippets of functionality. An Azure Function persists these code snippets and meta-information concerning when and how it should get executed. The function sleeps until it has been invoked by a trigger, which wakes it up, runs the code snippet and returns to sleep.
```

Comment:
I like this and the next part about scaling to 0. Also how Functions differs from web apis.